cmd/createorupdate/createorupdate.go

```diff
@@ -301,7 +301,8 @@ func main() {
 		return
 	}
 
-	if isCreate {
+	if _, err := os.Stat(path.Join(outputDirectory, "manifest.yaml")); os.IsNotExist(err) {
```

Comment:
This is simpler than a GET but if we want to simulate the real RP as much as possible we should be using the osa sdk to check that the openshift object actually exists. It may be problematic to do though since we don't persist objects on the fake rp side so running a hack/create.sh followed by a hack/upgrade.sh is not going to work as is.

Reply:
I guess that just felt like overkill @jim-minter what do you think?

Reply:
i think it should be the same way as: https://github.com/openshift/openshift-azure/blob/master/pkg/fakerp/fakerp.go#L47-L63
otherwise we could end up with one thinking it's an update and the other a create.